tests/Unit/CheckJwtMiddlewareTest.php

```diff
@@ -0,0 +1,260 @@
+<?php
+
+
+namespace Marketredesign\MrdAuth0Laravel\Tests\Unit;
+
+use Auth0\Login\Auth0JWTUser;
+use Illuminate\Foundation\Application;
+use Illuminate\Http\Request;
+use Marketredesign\MrdAuth0Laravel\Http\Middleware\CheckJWT;
+use Marketredesign\MrdAuth0Laravel\Tests\TestCase;
+use Symfony\Component\HttpKernel\Exception\HttpException;
+
+class CheckJwtMiddlewareTest extends TestCase
+{
+    /**
+     * Define environment setup.
+     *
+     * @param  Application  $app
+     * @return void
+     */
+    protected function getEnvironmentSetUp($app)
```

Comment:
Same as in other test, this doesn't return anything so not a getter

Reply:
Oh perhaps you mean it as like "Hey, can you get the environment set up?", so not a getter but get as in getting/doing/making? Either way, confusing.

Reply:
I fully agree, i have no idea what there were thinking when creating this method. Unfotunately nothing we can do about it though :(